### ISSUE TITLE
train musiq

### ISSUE THREAD

**Issue by HIT-Wang** [chaofengc/IQA-PyTorch#49]
hi,chaofeng,i try to retrain musiq-spaq model，but the result on the test set is quite different from the pretrain model pth you gave.Can you tell me some tricks or details when training musiq...🥹？such as learning rate? Or are you training the musiq-spaq model on top of other pretrained models?

**Comment by chaofengc** [comment]
The provided model is directly transfer from tensorflow checkpoints of the official weights: https://github.com/google-research/google-research/tree/master/musiq. 

The model is indeed training with pretrained models using imagenet, we also provided it here:
https://github.com/chaofengc/IQA-PyTorch/blob/3619109b4491d685837a795549db1f948fc19cc6/pyiqa/archs/musiq_arch.py#L27-L29

For other training details, please read the original paper in detail.

**Comment by HIT-Wang** [comment]
> The provided model is directly transferred from tensorflow checkpoints of the official weights: https://github.com/google-research/google-research/tree/master/musiq.
> 
> The model is indeed trained with pretrained models using imagenet, we have provided it here:
> 
> https://github.com/chaofengc/IQA-PyTorch/blob/3619109b4491d685837a795549db1f948fc19cc6/pyiqa/archs/musiq_arch.py#L27-L29
> 
> For other training details, **please read the original paper carefully.**

okay,thanks~

**Comment by ZachL1** [comment]
Hi, @HIT-Wang 

Have you successfully trained MUSIQ and verified its performance? Could you please share your training code? or configuration file if you implement it based on IQA-Pytorch.

Much appreciated!
